content/docs/tasks/traffic-management/egress/index.md

```diff
@@ -52,30 +66,38 @@ you cannot use this approach for services using those ports.
 
     {{< /tip >}}
 
-1.  Make a couple of requests to external HTTPS services from `SOURCE_POD`:
+1.  Make a couple of requests to external HTTPS services from `SOURCE_POD` and confirm that they produce
+    successful `200` responses:
 
     {{< text bash >}}
-    $ kubectl exec -it $SOURCE_POD -c sleep -- curl -s https://www.google.com | grep -o "<title>.*</title>"; kubectl exec -it $SOURCE_POD -c sleep -- curl -s https://edition.cnn.com | grep -o "<title>.*</title>"
-    <title>Google</title>
```

Comment:
I thought it is a nice visual clue. It also shows that each request arrived to its intended destination, and not to some other service that mistakenly returned 200.

Reply:
`grep -o` doesn't work so well no my machine. It doesn't seem to add a <CR> character so the output keeps moving to the right. The 200 codes print nice and clean and I think it's also very clear that the external request is going out